x-pack/metricbeat/module/awsfargate/_meta/docs.asciidoc

```diff
@@ -48,3 +48,17 @@ Currently, we have `task_stats` metricset in `awsfargate` module.
 This metricset collects runtime CPU metrics, disk I/O metrics, memory metrics,
 network metrics and container metadata from both endpoint
 `${ECS_CONTAINER_METADATA_URI_V4}/task/stats` and `${ECS_CONTAINER_METADATA_URI_V4}/task`.
+
+
+[float]
+== How to monitor containers running on ECS EC2
```

Comment:
Thanks for working on this!! Maybe we should add this section inside https://github.com/elastic/beats/pull/22784/files#diff-e1dda61aad4c73e0d21e21daf4f4c8f256e43a7cfda4ecedd2ffd02f239da959R21 so we only have one place mentioning ECS EC2 in `awsfargate` documentation.